Controllers/RoomsController.cs

```diff
@@ -1,46 +1,41 @@
-﻿using System;
+﻿using AsyncInn.Data.Interfaces;
+using AsyncInn.Models;
+using Microsoft.AspNetCore.Mvc;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.EntityFrameworkCore;
-using AsyncInn.Data;
-using AsyncInn.Models;
-using AsyncInn.Data.Interfaces;
 
 namespace AsyncInn.Controllers
 {
     [Route("api/[controller]")]
+    //[Route("{roomId}/Amenity/{amenityId}")]  POST
+    //[Route("{roomId}/Amenity/{amenityId}")]  DELETE
```

Comment:
These route values look good, but they would live on the individual actions, e.g.

```csharp
[HttpPost("{roomId}/Amenity/{amenityId}")]
```